common/types/message/message.go

```diff
@@ -52,22 +52,14 @@ type ChunkTaskDetail struct {
 
 // BatchTaskDetail is a type containing BatchTask detail.
 type BatchTaskDetail struct {
-	ChunkInfos      []*ChunkInfo     `json:"chunk_infos"`
-	ChunkProofs     []*ChunkProof    `json:"chunk_proofs"`
-	ParentStateRoot common.Hash      `json:"parent_state_root"`
-	ParentBatchHash common.Hash      `json:"parent_batch_hash"`
-	BatchHeader     *codecv3.DABatch `json:"batch_header"`
+	ChunkInfos  []*ChunkInfo     `json:"chunk_infos"`
+	ChunkProofs []*ChunkProof    `json:"chunk_proofs"`
+	BatchHeader *codecv3.DABatch `json:"batch_header"`
 }
 
 // BundleTaskDetail consists of all the information required to describe the task to generate a proof for a bundle of batches.
 type BundleTaskDetail struct {
-	ChainID             uint64        `json:"chain_id"`
-	FinalizedBatchHash  common.Hash   `json:"finalized_batch_hash"`
-	FinalizedStateRoot  common.Hash   `json:"finalized_state_root"`
-	PendingBatchHash    common.Hash   `json:"pending_batch_hash"`
-	PendingStateRoot    common.Hash   `json:"pending_state_root"`
-	PendingWithdrawRoot common.Hash   `json:"pending_withdraw_root"`
-	BatchProofs         []*BatchProof `json:"batch_proofs"`
+	BatchProofs []*BatchProof `json:"batch_proofs"`
```

Comment:
ditto?

Reply:
My initial understanding was that we might require those fields.

However, those fields are obtained from the "public input" of the `BatchCircuit` to add relevant constraints. Hence strictly from proof generation point-of-view, we do not really need those arguments. Hence I have removed them so that the coordinator does not need to fetch/prepare fields we don't really need.